frontend/src/components/input.tsx

```diff
@@ -60,9 +72,14 @@ export const Input = ({ sendMessage, waiting, suggestions }: InputProps) => {
     setUploadInProgress(true);
 
     try {
-      const { filename, id } = await uploadFileToServer(file);
-      console.log(`File uploaded successfully: ${filename} with id ${id}`);
+      const { filename, report } = await uploadFileToServer(file);
       setUploadedFile(file);
+      appendMessage(
+        { answer: `Your ESG report for ${filename} is ready to view.` },
+        Role.Bot,
+        report,
+        filename,
```

Comment:
Instead of passing in `filename`, can we pass in `` `ESG Report - ${filename}` ``

Reply:
This will allow us to pass in a different title for the data grid, rather than having the component always add `ESG Report -` in front of every sidePanelTitle

Reply:
Yeah, that makes sense. Will change now